### ISSUE TITLE
ctrl + <somekey> does not work on my Linux/KDE.

### ISSUE THREAD

**Issue by wyattbiker** [aseprite/aseprite#4263]
Press + <somekey> does not work on my system. As a matter of fact as soon as I press CTRL I get the Auto Select Layer choice in the tool bar. So for example I cannot perform an Undo or Copy/Paste. Only works by clicking on the menus.

I've reported this in the communities support but no one replied. I will assume it's some bug.

### Aseprite and System version
Asprite V1.3.2 x64 installed from HumbleBundle

Operating System: Kubuntu 23.10
KDE Plasma Version: 5.27.8
KDE Frameworks Version: 5.110.0
Qt Version: 5.15.10
Kernel Version: 6.5.0-14-generic (64-bit)
Graphics Platform: X11
Processors: 16 × 12th Gen Intel® Core™ i7-1260P
Memory: 15.3 GiB of RAM

**Comment by wyattbiker** [comment]
Hello, So I switched to another plain user. And it works! There must be some weird key assigned to CTRL key. Any suggestions how I can find out what the CTRL key does behind the scenes that it interferes with Asprite? Seems only Aseprite is affected by this. Al my other apps e.g. Krita, Gimp, text editors, etc work without a problem.

**Comment by martincapello** [comment]
Could you try opening "Edit > Keyboard Shortcuts..." and then click on any "Add" button on the "Key" column. This will show the "Keyboard Shortcut" dialog, then check the "Ctrl" checkbox. What does say the text next to the "Assigned to:" label?

**Comment by wyattbiker** [comment]
It says Assigned to Move Tool (quick).

**Comment by quardbreak** [comment]
Hey, I think I've found a clue what might be causing this, at least on KDE Plasma. It seems that Aseprite keyboard shortcuts do not work if the main keyboard layout is not "English (US)". I've flipped layouts, and now all the keyboard shortcuts work flawlessly!

**Comment by wyattbiker** [comment]
Yes absolutely fixes it!! I have multilingual keyboard in Settings > Keyboard > Layout. My English (US) keyboard was listed 2nd and moved it up to first. Now everything works. I hope Aseprite takes note and looks into it as I dont have this issue with any other applications. Thanks so much.

**Comment by shayaharuno** [comment]
@wyattbiker thank you very much. I've been struggling for a whole day with this weird issue, switching US layout as primary helped immediately.

**Comment by wyattbiker** [comment]
> @wyattbiker thank you very much. I've been struggling for a whole day with this weird issue, switching US layout as primary helped immediately.

@quardbreak is the one who turned me on to it. So shout out to them too :)

**Comment by DaRealNim** [comment]
> Hey, I think I've found a clue what might be causing this, at least on KDE Plasma. It seems that Aseprite keyboard shortcuts do not work if the main keyboard layout is not "English (US)". I've flipped layouts, and now all the keyboard shortcuts work flawlessly!
@quardbreak 

Thanks! That fixed it for me, as I had the opposite problem: I have a physical french AZERTY keyboard, and US QWERTY layout was in first position in my settings, so aseprite was using the QWERTY layout for shortcuts. Flipping it around solved the issue!

Though, the only reason I had QWERTY as a first layout is because when playing games with Steam's Proton, like with aseprite, it only uses the first keyboard layout, which is a problem for me in hardcoded QWERTY games 😅... I wish I could have default AZERTY and switch to QWERTY to play games that only support that, but that doesn't work.

Anyone knows if this a problem specific to KDE plasma? It's pretty annoying that I'd have to constantly change the default layout when switching between asprite and QWERTY-only games

**Comment by Overkane** [comment]
> Hey, I think I've found a clue what might be causing this, at least on KDE Plasma. It seems that Aseprite keyboard shortcuts do not work if the main keyboard layout is not "English (US)". I've flipped layouts, and now all the keyboard shortcuts work flawlessly!

Thanks! This fix with swapping keyboard layouts worked for me too. I was using steam version and my specs:
Aseprite version: 1.3.14-2-x64
Operating System: EndeavourOS 
KDE Plasma Version: 6.3.5
KDE Frameworks Version: 6.15.0
Qt Version: 6.9.1
Kernel Version: 6.15.2-arch1-1 (64-bit)
Graphics Platform: Wayland

Also a few other things I tried:
- installing from AUR - doesn't work, same issue like in steam version;
- forcing proton 9 and 10 in steam compatibility options - works, but proton's file manager makes it kinda unusable.

**Comment by crashtestjava** [comment]
I was able to reproduce this on Linux Ubuntu. Using certain keyboard layouts other than "English (US)" as the primary keyboard layout causes the issue. Some layouts other than English do not have the issue; other layouts do have the issue.

```
Aseprite Version: 1.3.15.4 x64
OS: Ubuntu 24.04.4 LTS, GNOME UI
Kernel: Linux 6.14.0-33-generic
```
